portfolio/src/test/java/com/google/sps/servlets/JournalServlet.java

```diff
@@ -1,19 +1,5 @@
-// Copyright 2019 Google LLC
```

Comment:
Is your test testing this file or the one in your source code?  You should be testing the same code that's used in production

Reply:
I've been currently making the code work here, then pasting it into the real file once it pasts the tests, I'm not sure exactly how to call the real functions from TestingTest.java

Reply:
You should just be able to import the actual JournalServlet class into your test.  This is fine for debugging purposes, but you shouldn't check this in.  Right now you're only testing a copy of the actual code, not the code itself